package.json

```diff
@@ -704,6 +704,11 @@
             "type": "boolean",
             "default": false,
             "markdownDescription": "Allow older schema versions to be used when producing messages to Kafka topics.\n\n- Enabling this will require selecting the schema version to use.\n\n- Disabling this will use the **latest schema version**.\n\n---\n\n⚠️ **WARNING**: Producing messages with older schema versions can be in violation of some schema compatibility settings."
+          },
+          "confluent.krb5ConfigPath": {
+            "type": "string",
+            "default": "",
+            "markdownDescription": "Path to the `krb5.conf` file to use for Kerberos authentication in direct connections. This setting is used when connecting to Kafka clusters that require Kerberos authentication."
```

Comment:
This should also come with a command to drive the OS file picker to choose the file and then set the command, like is done for PEM file paths and "Add SSL/TLS PEM file path". Hoomans aren't expected to know how to copy/paste paths anymore. 

But then again, are Kerberos users human?

Either fix here in branch or make a 'before Current' followup ticket.